wiki/Gameplay/Game_modifier/Easy/en.md

```diff
@@ -14,6 +14,8 @@ tags:
 *For other uses, see [Easy (disambiguation)](/wiki/Disambiguation/Easy).*\
 *For the full list of all mods, see: [Game modifier](/wiki/Gameplay/Game_modifier)*
 
+*For the osu!(lazer) wiki, see: [Easy (lazer)](/wiki/Gameplay/Game_modifier/Lazer/Easy)*
```

Comment:
- should be stacked like so (removes visual gap between the lines)
- imo the wording for these kinds of notices should be more like this

```suggestion
*For the full list of all mods, see: [Game modifier](/wiki/Gameplay/Game_modifier)*\
*For the [osu!(lazer)](/wiki/Client/Release_stream/Lazer) version of this article, see: [Easy (lazer)](/wiki/Gameplay/Game_modifier/Lazer/Easy)*
```